src/BinaryParsers/ElfBinary/Dwarf/DwarfEnums.cs

```diff
@@ -183,17 +183,23 @@ public enum DwarfAttribute
         Sibling = 0x01,
         Location = 0x02,
         Name = 0x03,
+        Reserved0 = 0x4,
```

Comment:
>Reserved0 [](http://example.com/codeflow?start=8&length=9)

I populated more values here taken from the DWARF5 spec. None of this work proved to be essential to the actual bug